File: SPOJ/FCTRL.py
from sys import stdin, stdout
repeat = int(stdin.readline())
x = 0
out = 0
for i in range(repeat):
    tmp = int(stdin.readline())
    while tmp != 0:
        tmp //=5
        out +=tmp
    stdin.write(str(out)+"\n")
    out = 0

File: SPOJ/FCTRL.c
#include <stdio.h>
int main() {

    int repeat, innum, out, tmp;
    scanf("%i", &repeat);
    while(repeat--)
    {
        out = 0;
        scanf("%i", &innum);
        while(innum)
        {
            innum = innum/5;
            out = out + innum;
        }
        printf("%i\n", out);
    }
    return 0;
}
